### DISCUSSION TITLE
视频内显示时间在预览版下无法安装

### DISCUSSION THREAD

**Discussion by Alexius66** [the1812/Bilibili-Evolved#4398] [功能建议 / Ideas]
在分支为preview下一点击安装就会空白报错，要安装只能切换到master分支
![image](https://github.com/the1812/Bilibili-Evolved/assets/73510996/11948a8e-68b5-42b9-9708-9c3c8c8d385a)
另外这个插件的透明度数字设置好像也有点问题🤣
![image](https://github.com/the1812/Bilibili-Evolved/assets/73510996/c1a3b0d6-aa2b-4a78-935e-7199b7967ef9)

**Comment by the1812** [comment]
preview 的要翻墙

**Reply by Alexius66** [reply]
我这里是翻墙了的，其他的插件都可以正常安装
